ScienceJournal/UI/CropRangeViewController.swift

```diff
@@ -62,11 +60,12 @@ class CropRangeViewController: UIViewController, EditTimestampViewControllerDele
 
   /// Shows an alert that allows the user to manually input a timestamp for the given crop marker.
   ///
-  /// - Parameter marker: A crop marker type.
-  func showTimestampEditAlert(forCropMarkerType markerType: CropOverlayView.MarkerType) {
-    guard let trialCropRange = trialCropRange else {
-      return
-    }
+  /// - Parameters
+  ///   - trialCropRange: The trial's crop range.
+  ///   - markerType: A crop marker type.
+  func showTimestampEditAlert(forTrialCropRange trialCropRange: ChartAxis<Int64>,
+                              andCropMarkerType markerType: CropOverlayView.MarkerType) {
+    self.trialCropRange = trialCropRange
```

Comment:
Setting this here feels a little weird to me, but I can't think of a better way to do it.

Reply:
Not for this CL but: CropRangeViewController is very odd. Looking at this again it, it should ideally have one instance per presentation of the crop so you could just initialize and present at once, rather than initializing and calling the `show` methods repeatedly. It doesn't even have it's own view. Makes me think it should just be a coordinator object or combine with the EditTimestampViewController class which already fits the "one instance per action" pattern.